README.markdown

```diff
@@ -42,7 +42,6 @@ The current set of results that we care about are comparing the latency that a p
 The result of running a test creates a few files data/seen.txt and data/updated.txt  data/seen.txt contains the counts of events for different campaigns and time windows.  data/updated.txt is the latency in ms from when the last event was emitted to kafka for that particular campaign window and when it was written into Redis.
 
 ### References
-Sanket Chintapalli, Derek Dagit, Bobby Evans, Reza Farivar, Thomas Graves, Mark Holderbaugh, Zhuo Liu, Kyle Nusbaum, Kishorkumar Patil, Boyang Jerry Peng, Paul Poulosky.
-"Benchmarking Streaming Computation Engines: Storm, Flink and Spark Streaming. " 
-First Annual Workshop on Emerging Parallel and Distributed Runtime Systems and Middleware. IEEE, 2016.
+Sanket Chintapalli, Derek Dagit, Bobby Evans, Reza Farivar, Thomas Graves, Mark Holderbaugh, Zhuo Liu, Kyle Nusbaum, Kishorkumar Patil, Boyang Jerry Peng, Paul Poulosky. Please read the blog at https://yahooeng.tumblr.com/post/135321837876/benchmarking-streaming-computation-engines-at to understand the tests and benchmark design. Or web search for topic "Benchmarking Streaming Computation Engines: Storm, Flink and Spark Streaming. " First Annual Workshop on Emerging Parallel and Distributed Runtime Systems and Middleware. IEEE, 2016.
```

Comment:
I have made the change. Thanks for taking time to review and suggesting the reference link to this excellent white paper.